pages/api/metadata/index.tsx

```diff
@@ -31,24 +31,15 @@ export default async function handler(req: any, res: any) {
   const metadata = response[`${chainId || CHAIN_ID}`]
   const finalAttribute = metadata?.attributes || deterministicAttribute
   const pfp = metadata?.pfp
-  const sheet = metadata?.sheet
   const ss = metadata?.ss
 
-  const endpoint = `api/og?type=${sheet?.type || type}&skin=${sheet?.skin || skin}&acc=${
-    sheet?.acc || acc
-  }&eye=${sheet?.eye || eye}&hair=${sheet?.hair || hair}&color=${sheet?.color || color}&outfit=${
-    sheet?.outfit || outfit
-  }&bg=${sheet?.bg || bg}&tokenId=${tokenId}`
-
-  const pfpUrl = pfp ? getIpfsLink(pfp) : `${DOMAIN_URL}/${endpoint}`
-  const sheetUrl = ss
-    ? getIpfsLink(ss)
-    : `${DOMAIN_URL}/spritesheet/${chainId || CHAIN_ID}/${tokenId}`
+  const pfpUrl = pfp ? getIpfsLink(pfp) : `${DOMAIN_URL}/api/image/${tokenId}`
+  const sheetUrl = ss ? getIpfsLink(ss) : `${DOMAIN_URL}/api/spritesheet/${tokenId}`
```

Comment:
we do not want to return `ipfs` hashes in the description
- actual: showing IPFS hash in description
- expected: always show `api/` route for pfp / spritesheet

also need to make sure the `/api` endpoint you pick is actually returning the `image` you expect it to return

I don't see that included in this PR
- am I missing something?